notification/src/main/java/com/caldevsupplychain/notification/mail/service/EmailServiceImpl.java

```diff
@@ -55,6 +75,26 @@ public void sendMimeMessage(EmailTemplate emailTemplate) throws MessagingExcepti
 		helper.setTo(emailTemplate.getToEmail());
 		helper.setFrom(emailTemplate.getFromEmail());
 		helper.setText(emailTemplate.getContent(), true);
+
 		javaMailSender.send(message);
 	}
+
+	public String getRootURL(){
+		InetAddress ip;
+		String root = null;
+
+		try {
+			ip = apiMode.equals("dev") ? InetAddress.getLoopbackAddress() : InetAddress.getLocalHost();
```

Comment:
https://docs.oracle.com/javase/7/docs/api/java/net/InetAddress.html#getAllByName(java.lang.String)
Looks like this handles IP address. it won't handle url. 
Try `ServletUriComponentsBuilder` https://stackoverflow.com/questions/1490821/whats-the-best-way-to-get-the-current-url-in-spring-mvc

Reply:
fix